Cargo.toml

```diff
@@ -18,6 +18,7 @@ path = "src/cli.rs"
 [dependencies]
 anyhow = "1.0.75"
 base16ct = "0.2.0"
+chrono = "0.4"
```

Comment:
Adding this dep in preparation for better handling of DateTimes, might as well get it set up internally